### PR TITLE
Create a listing page

### PR DESCRIPTION
**Pull request by tobygabriella**
https://github.com/tobygabriella/worldcollective/assets/136364857/4ede0894-fb10-4eeb-90ff-8e97eea48384

### CODE REVIEW

File: Frontend/src/App.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Home = lazy(() => import('./Components/Home/Home.jsx'));
 const LogIn = lazy(() => import('./Components/Login/LogIn.jsx'));
 const SignUp = lazy(() => import('./Components/Signup/SignUp.jsx'));
 const UserProfile = lazy(()=>import('./Components/UserProfile/UserProfile.jsx'))
-
+const CreateListing =lazy(()=>import('./Components/CreateListing/CreateListing.jsx'))
```
Comment (conniewu3) [inline Frontend/src/App.jsx]: nit: add a space between = and lazy

File: Frontend/src/Components/CreateListing/InfoSection.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <div className="infoSection">
+      <h3>Info</h3>
+      <label>
+        Category
+        <select value={category} onChange={(e) => setCategory(e.target.value)}>
+            <option value="" disabled>Select a category</option>
+            <option value="clothing">Clothing</option>
+            <option value="accessories">Accessories</option>
+            <option value="accessories">Furniture</option>
```
Comment (conniewu3) [inline Frontend/src/Components/CreateListing/InfoSection.jsx]: are these values all supposed to be accessories?

File: Frontend/src/Components/CreateListing/PhotoSection.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+const PhotoSection = ({ photos, handlePhotoUpload, handlePhotoDelete }) => {
+  return (
+    <div className="photosSection">
+      <h3>Photos</h3>
+      <p>Add up to 8 photos in JPEG or PNG format</p>
+      <div className="photosGrid">
+        {Array(8)
+          .fill()
+          .map((_, index) => (
```
Comment (conniewu3) [inline Frontend/src/Components/CreateListing/PhotoSection.jsx:12]: nice job using a map!

File: Frontend/src/Components/Home/Home.jsx [hunk 4]
```diff
@@ -14,11 +14,6 @@ const Home = () => {
     }
   };
 
-  const handleLogout = () => {
```
Comment (conniewu3) [inline Frontend/src/Components/Home/Home.jsx:17]: why did you remove this?

**Comment by conniewu3** [comment]
a few small things but otherwise this looks good!

**Review by conniewu3 — APPROVED** [review]

File: Frontend/src/Components/Home/Home.jsx (hunk 4 above)
Comment (tobygabriella) [reply Frontend/src/Components/Home/Home.jsx:17]: I moved the function to my header because thats where my logout button is

File: Frontend/src/Components/CreateListing/InfoSection.jsx (hunk 2 above)
Comment (tobygabriella) [reply Frontend/src/Components/CreateListing/InfoSection.jsx]: No, but ive made the changes!
